handler.py

```diff
@@ -0,0 +1,94 @@
+
+import os
+import logging
+import json
+import urllib.parse
+
+from rdflib.plugins.sparql.parser import parseUpdate
+from rdflib.plugins.sparql.algebra import translateUpdate
+
+from sns_client import SnsClient
+from neptune_client import NeptuneClient
+
+
+def main(event, _):
+    # Setup the logger at the INFO level while we continue to profile
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+
+    rialto_sparql_endpoint = os.getenv('RIALTO_SPARQL_ENDPOINT', "http://localhost:8080/bigdata/namespace/kb/sparql")
+    rialto_sns_endpoint = os.getenv('RIALTO_SNS_ENDPOINT', "http://localhost:4575")
+    rialto_topic_arn = os.getenv('RIALTO_TOPIC_ARN', "rialto")
+    aws_region = os.getenv('AWS_REGION', "us-west-2")
+
+    sns_client = SnsClient(rialto_sns_endpoint, rialto_topic_arn, aws_region)
+    neptune_client = NeptuneClient(rialto_sparql_endpoint)
+
+    response, status_code = neptune_client.post(event['body'])
+
+    if "update=" in event['body'] and status_code == 200:
```

Comment:
Why does `update=` have to be in event body? I don't think we add it when posting from ETL.

Reply:
I would assume it gets added as part of the query body otherwise neptune would complain about it. An error is returned without it. In this case, we don't want to bother with the parsing if it's simply a query and not an update as we don't have any reason to put a message on the sns topic.

Reply:
In etl, we set `'Content-Type' => 'application/sparql-update'`. Does that take the place of `update=`?

Reply:
That dawned on my during my walk, I'm wondering if we're missing some messaging because of that? Will test.